sessionctx/variable/session.go

```diff
@@ -417,6 +417,9 @@ type SessionVars struct {
 
 	// PrevStmt is used to store the previous executed statement in the current session.
 	PrevStmt string
+
+	// AllowDropAutoInc indicates whether a user can drop the auto_increment column attribute or not.
+	AllowDropAutoInc bool
```

Comment:
```suggestion
	AllowRemoveAutoInc bool
```
IMO using `Remove` is better, `Drop` reminds the users that data are dropped(like `DROP TABLE` or `DROP COLUMN`, but this is not the case)